index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ekpenisi E Raphael portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@500&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <nav id="nav-bar">
+          <label id="logo">Ekpenisi E Raphael</label>
```

Comment:
- [ ] [OPTIONAL] Great job implementing this project. But the logo should be wrapped within an `<a>` tag because it is best practice to have the logo clickable and link to the home page of a website. It is optional because the previous reviewer omitted it but it is highly recommended that you implement this.